### ISSUE TITLE
Pls update

### ISSUE THREAD

**Issue by rube200** [Senior-S/MyVehicleKeys#1]
Please update the plugin but use datastore or userdata instead of direct access to path
https://github.com/Senior-S/MyVehicleKeys/blob/bf924dff2e7fffc75b36c9cb9413076f4a44745c/Utils.cs#L151
```
[2020-11-16 00:00:19 INF][MyVehicleKeys.MyVehicleKeys] [loading] MyVehicleKeys v1.0.1
[2020-11-16 00:00:19 ERR][OpenMod.Core.Plugins.PluginActivator] Failed to load plugin: MyVehicleKeys v1.0.1
System.IO.DirectoryNotFoundException: Could not find a part of the path "C:\Unturned-Server\Servers\SuperSv\Rocket\Servers\SuperSv\OpenMod\plugins\SS.MyVehicleKeys\PlayerKeys.json".
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x00164] in <fb001e01371b4adca20013e0ac763896>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options, System.String msgPath, System.Boolean bFromProxy, System.Boolean useLongPath, System.Boolean checkHost) [0x00000] in <fb001e01371b4adca20013e0ac763896>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream..ctor(string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare,int,System.IO.FileOptions,string,bool,bool,bool)
  at System.IO.StreamWriter.CreateFile (System.String path, System.Boolean append, System.Boolean checkHost) [0x0001c] in <fb001e01371b4adca20013e0ac763896>:0 
  at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding, System.Int32 bufferSize, System.Boolean checkHost) [0x00055] in <fb001e01371b4adca20013e0ac763896>:0 
  at System.IO.StreamWriter..ctor (System.String path, System.Boolean append, System.Text.Encoding encoding, System.Int32 bufferSize) [0x00000] in <fb001e01371b4adca20013e0ac763896>:0 
  at System.IO.StreamWriter..ctor (System.String path, System.Boolean append) [0x00008] in <fb001e01371b4adca20013e0ac763896>:0 
  at (wrapper remoting-invoke-with-check) System.IO.StreamWriter..ctor(string,bool)
  at System.IO.File.AppendAllText (System.String path, System.String contents) [0x00000] in <fb001e01371b4adca20013e0ac763896>:0 
  at MyVehicleKeys.Utils.CreateInitialFile () [0x00042] in <c8767f9aacae4d4a904f24bec9e6d756>:0 
  at MyVehicleKeys.MyVehicleKeys+<OnLoadAsync>d__6.MoveNext () [0x0004b] in <c8767f9aacae4d4a904f24bec9e6d756>:0 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <fb001e01371b4adca20013e0ac763896>:0 
  at Cysharp.Threading.Tasks.UniTask+ExceptionResultSource.GetResult (System.Int16 token) [0x00000] in <1ff745593d514fdcbaa381ce7274595e>:0 
  at OpenMod.UnityEngine.Plugins.OpenModUnityEnginePlugin+<LoadAsync>d__1.MoveNext () [0x0015e] in <91469cb2870748ca8b8a602993f49524>:0 
--- End of stack trace from previous location where exception was thrown ---
  at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw () [0x0000c] in <fb001e01371b4adca20013e0ac763896>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Threading.Tasks.Task task) [0x0003e] in <fb001e01371b4adca20013e0ac763896>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Threading.Tasks.Task task) [0x00028] in <fb001e01371b4adca20013e0ac763896>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd (System.Threading.Tasks.Task task) [0x00008] in <fb001e01371b4adca20013e0ac763896>:0 
  at System.Runtime.CompilerServices.TaskAwaiter.GetResult () [0x00000] in <fb001e01371b4adca20013e0ac763896>:0 
  at OpenMod.Core.Plugins.PluginActivator+<TryActivatePluginAsync>d__11.MoveNext () [0x0036e] in <24148c8d84b040faa42f4ac5493d8aa0>:0 
```

**Comment by Senior-S** [comment]
Hello, ok, I'm currectly with exams and a lot of homework, when i finish all of this i will update all of my plugins with better practices.

**Comment by Senior-S** [comment]
Already fixed in the 1.2.5
